src/tapis-ui/_common/Breadcrumbs/Breadcrumbs.test.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import renderComponent from 'utils/testing';
+import Breadcrumbs from 'tapis-ui/_common/Breadcrumbs';
+import breadcrumbsFromPathname from './breadcrumbsFromPathname';
+
+const _4_BREADCRUMBS = breadcrumbsFromPathname("/files/system/a/b/")
+const _5_BREADCRUMBS = breadcrumbsFromPathname("/files/system/a/b/c/")
+
+describe('Breadcrumbs', () => {
+  it('renders Breadcrumbs without an ellipsis', () => {
+    const { queryAllByText } = renderComponent(
```

Comment:
There should be a unit test for 1 and 3 breadcrumbs, and the tests should also check to see that the text is rendered for those breadcrumbs.

Reply:
Added tests for 1 and 3 breadcrumbs and tested for correct text in component.